index.html

```diff
@@ -7,10 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+  <link rel="StyleSheet" href="css/style.css">
 </head>
 
+
 <body>
-    <!-- Add your markup here -->
+    <header>  
+        <img class="logo-image" src="Images/logo.jfif" alt="logo">
+        <p class="hamburger">&#9776</p>
+        <p class="header-p">The best cakes in town delivered to your door</p>
+    </header>
+    <main class="main">
+        <div class="welcome">
+            <nav class="nav">
+                <a class="nav-a" href="">HOME</a>
+                <a class="nav-a" href="">CAKES</a>
+                <a class="nav-a" href="">ORDERIN</a>
+                <a class="nav-a" href="">LESSONS</a>
+                <a class="nav-a" href="">ABOUT</a>
+            </nav>
+            <h2 class="intro-title">Welcome</h2>
+            <p  class="intro-p">here are many variations of passages of Lorem Ipsum 
+                available, but the majority have suffered alteration 
+                in some form, by injected humour, or randomised words
+                which don't look even slightly believable. If you are 
+                going to use a passage of Lorem Ipsum, you need to be 
+                sure there isn't anything embarrassing hidden in the middle of text. 
+                All the Lorem Ipsum.</p>
+            <img class="intro-img" src="Images/Chocolate-Cake.jpg" alt="Chocolate Cake"> 
+        </div>
+
+        <div class="options">
+            <img class="cake1" class="option-cakes" src="Images/fruite-Cake.jpg" alt="fruite Cake">
+            <img class="cake2"class="option-cakes" src="Images/Lemon-Cake.jpg" alt="Lemon Cake">
+            <img class="cake3" class="option-cakes" src="Images/Rainbow-Cake.jpg" alt="Rainbow Cake">
+            <img class="cake4" class="option-cakes" src="Images/Cake.jpg" alt="Cake">
+        </div>
+    </main>
+
+ 
+    <footer class="footer">
+        <a class="footer-link" href="" target="_blank">Facebook</a>
+        <a class="footer-link" href="" target="_blank">Twitter</a>
+        <a class="footer-link" href="" target="_blank">Instagram</a>
+        <p class="footer-hamburger">&#9776</p>
+        <P>You can follow us on</P>
```

Comment:
These links aren't appearing on mobile, and on desktop they're appearing before the 'You can follow us on'